src/main/java/com/example/balanceGame/controller/BoardController.java

```diff
@@ -25,8 +27,16 @@ public ResponseEntity regist(@RequestBody BoardRegistRequest boardRegistRequest,
         return boardService.regist(boardRegistRequest, principal);
     }
 
+    // 게시글 상세조회 controller
     @GetMapping("/detail")
     public ResponseEntity<BoardDetailResponse> detail(@RequestParam("boardKey") long boardKey) {
         return boardService.findBoardProfile(boardKey);
     }
+
+    // 게시글 페이징 날짜순으로 조회
+    @GetMapping("/findAllByDate")
+    public ResponseEntity<FindAllByDateResponse> findAllByDate(@RequestParam("page") Integer page) {
+        PageRequest pageRequest = PageRequest.of(page, 20);
```

Comment:
```suggestion
    public ResponseEntity<FindAllByDateResponse> findAllByDate(
        @RequestParam("page") Integer page,
        @RequestParam("size", defaultValue="20", required=false) Integer size,
    ) {
        PageRequest pageRequest = PageRequest.of(page, size);
```

나였다면 size 도 받아서 클라에서 유동적으로 관리하도록 할 듯 

페이징 관련은 대부분 이렇게 작업함

Reply:
굳이 받아야할까? 해서 기본으로 준건데 클라이언트 관점에서 생각해야하는군요 알겠습니다